app/src/main/java/edu/uw/tcss450/chatapp/ui/weather/WeatherListFragment.java

```diff
@@ -39,5 +60,10 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                 Navigation.findNavController(getView()).navigate(
                         WeatherListFragmentDirections.actionNavigationWeatherToWeatherMapFragment())
         );
+
```

Comment:
You can remove these lines

Reply:
The app won't navigate to the WeatherMapFragment then